src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -62,6 +67,20 @@ private static void setupCommandTests(Application application) {
         application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
 
         application.addTest("Count subcommands", (e) -> CountCommandsTest.execute());
+
+        TransformCurrentCommandOptionListener rot45flipX = new TransformCurrentCommandOptionListener(Arrays.asList(
+                new RotateTransformation(45),
+                new FlipTransformation(true, false)
+        ));
+
+        application.addTest("Transform: Rotate 45 & Flip X", rot45flipX);
```

Comment:
What would you prefer in Paint application: to have "Rotate 45 & Flip X" operation or separate Rotate 45 and Flip X operations?